tests/test_transformers.py

```diff
@@ -42,7 +42,7 @@
     Normalizer(),
     Normalizer(feature=True),
 ]
-extra_sizes = [2, 3, 2, 3, 0, 0, 4, 1, 0, 2, 3, 2, 2, 3, 0, 0]
+extra_sizes = [0, 0, 0, 0, 0, 0, 4, 1, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
To be honest. I'm open to deprecating `AddRandom`. It feels like that operation should be done via "make a new embeddingset with random vectors and concatenate". Might be best for another PR but wanted to mention it.

Reply:
All right! Though, since I can't think of a good use case for it, I won't comment further.